client/header/activity-panel/panels/orders.js

```diff
@@ -266,6 +266,7 @@ export default compose(
 			getReportItemsError,
 			isReportItemsRequesting,
 		} = select( 'wc-api' );
+		wcSettings.wcAdminSettings = wcSettings.wcAdminSettings || {};
```

Comment:
Is the plan to remove this and use `getSetting` in this file too?

Reply:
Yes.  I think in an initial pass I was looking at leaving some of the "legacy" way of doing things intact so I could do more granular pulls for other refactors in further iterations (this pull was already getting pretty big).  I probably just noticed this global wasn't initialized with a default in this file and added this in.

Reply:
https://github.com/woocommerce/woocommerce-admin/issues/2943